### PR TITLE
Aducm makefile flags fix

### PR DESCRIPTION
**Pull request by adrimbarean**
A couple of fixes for Windows building.

### CODE REVIEW

**Review by amiclaus — APPROVED** [review]

File: tools/scripts/generic.mk [hunk 1]
```diff
@@ -221,6 +221,7 @@ endif
 #Add to include all directories containing a .h file
 EXTRA_INC_PATHS += $(sort $(foreach dir, $(INCS_IN_BUILD),$(dir $(dir))))
 CFLAGS += $(addprefix -I,$(EXTRA_INC_PATHS) $(PLATFORM_INCS))
+CFLAGS += $(VERB_INC_FLAGS)
```
Comment (mchindri) [inline tools/scripts/generic.mk:227]: nitpick: I think is cleaner if you just add -I to all PLATFORM_INCS and 
CFLAGS += $(PLATFORM_INCS)

**Comment by adrimbarean** [comment]
V1:
 - make PLATFORM_INCS directly with -I prefix and don't use addprefix for it.

File: tools/scripts/aducm.mk [hunk 2]
```diff
@@ -110,7 +110,8 @@ C_RELEASE_FLAGS = -O2
 
 CFLAGS += -DADUCM_PLATFORM
 
-ASFLAGS	+= $(GENERIC_FLAGS) $(addprefix -I,$(PLATFORM_INCS)) -x assembler-with-cpp 
+ASFLAGS	+= $(GENERIC_FLAGS) $(PLATFORM_INCS) -x assembler-with-cpp 
+ASFLAGS += $(VERB_INC_FLAGS)
```
Comment (mchindri) [inline tools/scripts/aducm.mk:114]: Now this line is not needed

File: tools/scripts/generic.mk [hunk 3]
```diff
@@ -218,6 +218,9 @@ ifneq ($(REL_ASM_SRCS),$(ASM_OBJS_S))
 	ASM_OBJS += $(ASM_OBJS_S)
 endif
 
+SRCS_IN_BUILD = $(call relative_to_project, $(SRCS))
```
Comment (mchindri) [inline tools/scripts/generic.mk:221]: I wanted to do this but this way:
image

else you have to move olaso INCS:= $(sort $(INCS)) and SRCS = $(sort $(SRCS))

**Comment by adrimbarean** [comment]
V2:
 - remove redundant `ASFLAGS += $(VERB_INC_FLAGS)`
 - fix `EXTRA_INC_PATHS` [the right way](https://github.com/analogdevicesinc/no-OS/pull/937#discussion_r576635673).

**Review by mchindri — APPROVED** [review]

**Pull request merged**